src/main/java/edu/ksu/canvas/attendance/controller/CourseConfigurationController.java

```diff
@@ -86,12 +87,33 @@ public ModelAndView classSetup(@PathVariable String sectionId,
         return page;
     }
 
+    private ModelAndView validateInputAsDouble(String sectionId, CourseConfigurationForm classSetupForm) {
+
+        if (classSetupForm.getGradingOn()){
+            boolean typeValidation = classSetupForm.getAssignmentPoints() instanceof Double && classSetupForm.getPresentPoints() instanceof Double && classSetupForm.getExcusedPoints() instanceof Double && classSetupForm.getAbsentPoints() instanceof Double && classSetupForm.getTardyPoints() instanceof Double;
+
+            if(!typeValidation){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "/courseConfiguration" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)